src/fromJS.js

```diff
@@ -36,5 +36,9 @@ function fromJSDefault(json) {
 }
 
 function isPlainObj(value) {
-  return value && (value.constructor === Object || value.constructor === undefined);
+  return value &&
+    (value.constructor === Object ||
+      value.constructor === undefined ||
+      Object.prototype.toString.call(value) === '[object Object]'
```

Comment:
slowest action in the end. For applications which have only one context it won't be a performance-breaker.